loader/convert_data.go

```diff
@@ -227,7 +228,7 @@ func tableName(schema, table string) string {
 	return fmt.Sprintf("`%s`.`%s`", schema, table)
 }
 
-func parseTable(r *router.Table, schema, table, file string) (*tableInfo, error) {
+func parseTable(logger log.Logger, r *router.Table, schema, table, file string) (*tableInfo, error) {
```

Comment:
put log in function's args seems ugly... have any other way?

Reply:
think logger as session context, in fact I develop a context pkg and still don't use it now, any suggestions?